test/utils/config/arbitrumMainnet.json

```
@@ -7,31 +7,11 @@
   "DAI_ADDRESS": "0x0000000000000000000000000000000000000000",
   "LON_ADDRESS": "0x0000000000000000000000000000000000000000",
   "WBTC_ADDRESS": "0x0000000000000000000000000000000000000000",
-  "ANKRETH_ADDRESS": "0x0000000000000000000000000000000000000000",
 
-  "UNISWAP_V2_ADDRESS": "0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D",
+  "CURVE_TRICRYPTO2_POOL_ADDRESS": "0x0000000000000000000000000000000000000000",
   "SUSHISWAP_ADDRESS": "0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506",
-  "UNISWAP_V3_ADDRESS": "0xE592427A0AEce92De3Edee1F18E0157C05861564",
   "UNISWAP_V3_QUOTER_ADDRESS": "0x0000000000000000000000000000000000000000",
   "UNISWAP_PERMIT2_ADDRESS": "0x000000000022d473030f116ddee9f6b43ac78ba3",
   "UNISWAP_SWAP_ROUTER_02_ADDRESS": "0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45",
-  "UNISWAP_UNIVERSAL_ROUTER_ADDRESS": "0x4648a43B2C14Da09FdF82B161150d3F634f40491",
-  "CURVE_USDT_POOL_ADDRESS": "0x0000000000000000000000000000000000000000",
-  "CURVE_COMPOUND_POOL_ADDRESS": "0x0000000000000000000000000000000000000000",
-  "CURVE_Y_POOL_ADDRESS": "0x0000000000000000000000000000000000000000",
-  "CURVE_3_POOL_ADDRESS": "0x0000000000000000000000000000000000000000",
-  "CURVE_TRICRYPTO2_POOL_ADDRESS": "0x0000000000000000000000000000000000000000",
-  "CURVE_ANKRETH_POOL_ADDRESS": "0x0000000000000000000000000000000000000000",
-  "BALANCER_V2_ADDRESS": "0x0000000000000000000000000000000000000000",
-
-  "cDAI_ADDRESS": "0x0000000000000000000000000000000000000000",
-  "cUSDC_ADDRESS": "0x0000000000000000000000000000000000000000",
-  "yDAI_ADDRESS": "0x0000000000000000000000000000000000000000",
-  "yUSDC_ADDRESS": "0x0000000000000000000000000000000000000000",
-  "yUSDT_ADDRESS": "0x0000000000000000000000000000000000000000",
-  "yTUSD_ADDRESS": "0x0000000000000000000000000000000000000000",
-
-  "ARBITRUM_L1_GATEWAY_ROUTER_ADDR": "0x0000000000000000000000000000000000000000",
-  "ARBITRUM_L1_BRIDGE_ADDR": "0x0000000000000000000000000000000000000000",
-  "OPTIMISM_L1_STANDARD_BRIDGE_ADDR": "0x0000000000000000000000000000000000000000"
+  "UNISWAP_UNIVERSAL_ROUTER_ADDRESS": "0x4648a43B2C14Da09FdF82B161150d3F634f40491"
```

```suggestion
  "UNISWAP_UNIVERSAL_ROUTER_ADDRESS": "0x4C60051384bd2d3C01bfc845Cf5F4b44bcbE9de5"
  "UNISWAP_UNIVERSAL_ROUTER_ADDRESS_V1_2": "0xeC8B0F7Ffe3ae75d7FfAb09429e3675bb63503e4"
```
Addresses should be these according to latest code: https://github.com/Uniswap/universal-router/blob/main/deploy-addresses/arbitrum.json

---

Fixed 22e1a5928e739ca5c844ca10b7f7ea745bafb9a8